easybuild/easyconfigs/u/unixODBC/unixODBC-2.3.7-foss-2019b.eb

```diff
@@ -0,0 +1,20 @@
+# Easyconfig for unixODBC
+# Author: Lykle Voort
+# SURFsara, Amsterdam, The Netherlands
+
+easyblock = 'ConfigureMake'
+
+name = 'unixODBC'
+version = '2.3.7'
+
+homepage = "https://www.unixodbc.org"
+description = """unixODBC provides a uniform interface between
+application and database driver"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
This easyconfig for unixODBC can be moved to the `GCCcore/8.3.0` toolchain as it does not require MPI or BLAS/LAPACK.

Reply:
module unixODBC/2.3.7-GCCcore-8.3.0 has been submitted